src/tests/work-item/work-item-list/test-template.spec.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Template for automated UI tests for ALMighty
+ *  
+ * Note on screen resolutions - See: http://www.itunesextractor.com/iphone-ipad-resolution.html
+ * Tests will be run on these resolutions:
+ * - iPhone6s - 375x667
+ * - iPad air - 768x1024
+ * - Desktop -  1920x1080
+ * 
+ * beforeEach will set the mode to phone. Any tests requiring a different resolution will must set explicitly. 
+ * 
+ * @author 
+ */
+
+var WorkItemListPage = require('./work-item-list.page'),
+  testSupport = require('./testSupport');
+
+describe('Test name here...', function () {
+  var page, items, browserMode;
+
+  beforeEach(function () {
+    testSupport.setBrowserMode('phone');
+    page = new WorkItemListPage();    
+  });
+
+  it('should testsomething here...', function() {
+//    expect(something...).toBe(comething...);
```

Comment:
is this an example or are you missing a test? a placeholder to be implemented later?